packages/browser/core/src/core/internal/net/BrowserFetchStreamTransport.ts

```diff
@@ -16,52 +16,55 @@
 
 import {
   BasicRequest,
+  ContentTypes,
   EventStream,
+  handleRequestError,
   Headers,
-  ContentTypes,
   Response,
-  Transport,
-  handleRequestError
+  Transport
 } from "mongodb-stitch-core-sdk";
+import {fetch as fetch} from 'whatwg-fetch'
 import BrowserFetchTransport from "./BrowserFetchTransport";
 import EventSourceEventStream from "./EventSourceEventStream";
-import {fetch as fetch} from 'whatwg-fetch'
 
 /** @hidden */
 export default class BrowserFetchStreamTransport extends BrowserFetchTransport {
 
-  public stream(request: BasicRequest, open: boolean = true, retryRequest?: () => Promise<EventStream>): Promise<EventStream> {
-    let headers = { ...request.headers };
-    headers[Headers.ACCEPT] = ContentTypes.TEXT_EVENT_STREAM;
-    headers[Headers.CONTENT_TYPE] = ContentTypes.TEXT_EVENT_STREAM;
+  public stream(request: BasicRequest, open = true, retryRequest?: () => Promise<EventStream>): Promise<EventStream> {
+    const reqHeaders = { ...request.headers };
+    reqHeaders[Headers.ACCEPT] = ContentTypes.TEXT_EVENT_STREAM;
+    reqHeaders[Headers.CONTENT_TYPE] = ContentTypes.TEXT_EVENT_STREAM;
 
     // Verify we can start a request with current params and potentially
-    // force ourselves to refresh a token.
+    // Force ourselves to refresh a token.
     return fetch(request.url + "&stitch_validate=true", {
       body: request.body,
-      headers: headers,
+      headers: reqHeaders,
       method: request.method,
       mode: 'cors'
     }).then(response => {
-      const headers: { [key: string]: string } = {};
+      const respHeaders: { [key: string]: string } = {};
       response.headers.forEach((value, key) => {
-        headers[key] = value;
+        respHeaders[key] = value;
       });
       if (response.status < 200 || response.status >= 300) {
         return response.text()
-        .then(body => handleRequestError(new Response(headers, response.status, body)));
+        .then(body => handleRequestError(new Response(respHeaders, response.status, body)));
       }
 
-      return new Promise<EventStream>((resolve, reject) => {
```

Comment:
Couldnt find a way around this without breaking things

Reply:
👍 just add a comment explaining why you disabled the rule since this is not test code